datahub-web-react/src/app/ingestV2/executions/components/SummaryTab.tsx

```diff
@@ -0,0 +1,209 @@
+        margin-bottom: 12px;
+    }
+`;
+
+const SectionSubHeader = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const ButtonGroup = styled.div`
+    display: flex;
+    gap: 8px;
+    align-items: center;
+`;
+
+const SubHeaderParagraph = styled(Typography.Paragraph)`
+    margin-bottom: 0px;
+`;
+
+const StatusSection = styled.div`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const IngestedAssetsSection = styled.div<{ isFirstSection?: boolean }>`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    ${({ isFirstSection }) => !isFirstSection && `border-top: 1px solid ${ANTD_GRAY[4]};`}
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const RecipeSection = styled.div`
+    border-top: 1px solid ${ANTD_GRAY[4]};
+    padding-top: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const LogsSection = styled.div`
+    padding-top: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const ShowMoreButton = styled(Button)`
+    padding: 0px;
+`;
+
+const DetailsContainer = styled.div<DetailsContainerProps>`
+    margin-bottom: -25px;
+    ${(props) =>
+        props.areDetailsExpandable &&
+        !props.showExpandedDetails &&
+        `
+        -webkit-mask-image: linear-gradient(to bottom, rgba(0,0,0,1) 50%, rgba(255,0,0,0.5) 60%, rgba(255,0,0,0) 90% );
+        mask-image: linear-gradient(to bottom, rgba(0,0,0,1) 50%, rgba(255,0,0,0.5) 60%, rgba(255,0,0,0) 90%);
+    `}
+`;
+
+type DetailsContainerProps = {
+    showExpandedDetails: boolean;
+    areDetailsExpandable: boolean;
+};
+
+export const SummaryTab = ({
+    urn,
+    status,
+    result,
+    data,
+    onTabChange,
+}: {
+    urn: string;
+    status: string | undefined;
+    result: Partial<ExecutionRequestResult>;
+    data: GetIngestionExecutionRequestQuery | undefined;
+    onTabChange: (tab: TabType) => void;
+}) => {
+    const [showExpandedLogs] = useState(false);
+    const [showExpandedRecipe] = useState(false);
+
+    const output = data?.executionRequest?.result?.report || 'No output found.';
+
+    const downloadLogs = () => {
+        downloadFile(output, `exec-${urn}.log`);
+    };
+
+    const logs = (showExpandedLogs && output) || output?.split('\n')?.slice(0, 14)?.join('\n');
+    const structuredReport = result && getStructuredReport(result);
+    const resultSummaryText =
+        (status && status !== EXECUTION_REQUEST_STATUS_SUCCESS && (
+            <Typography.Text type="secondary">{getExecutionRequestSummaryText(status)}</Typography.Text>
+        )) ||
+        undefined;
+    const recipeJson = data?.executionRequest?.input?.arguments?.find((arg) => arg.key === 'recipe')?.value;
+    let recipeYaml: string;
+    try {
+        recipeYaml = recipeJson && YAML.stringify(JSON.parse(recipeJson), 8, 2).trim();
+    } catch (e) {
+        recipeYaml = '';
+    }
```

Comment:
this recipe fetching and parsing could be a shared util since the standalone tab has the same needed as far as i can tell

Reply:
It could have been until we split the tabs into different files as per your earlier suggestion. Where would I put this?

Reply:
This can just be a utility that both components import.

Reply:
@anshbansal was this resolved?